### PR TITLE
Add documentation for new ASA Server update endpoints

### PR DESCRIPTION
**Pull request by phoebeyu-okta**
## Description:
- **What's changed?**: Added documentation for new Server Update endpoints under the ASA section. 
- **Is this PR related to a Monolith release?**: No

### Resolves:

* [OKTA-395544](https://oktainc.atlassian.net/browse/OKTA-395544)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md [hunk 1]
```diff
@@ -1739,6 +1740,7 @@ This endpoint returns a list of objects with the following fields and a `200` co
 | `hostname`   | string | The hostname of the Server |
 | `id`   | string | The UUID corresponding to the Server |
 | `instance_details`   | object | Information that the cloud provider provides about the Server, if one exists |
+| `labels`   | object | (Optional) The labels for this server. Only available with the PolicySync feature, which is currently in EA. |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md:1743]:
```suggestion
| `labels`   | object | (Optional) The labels for this server. This parameter is only available with the PolicySync feature, which is currently in EA. |
```

File: packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md [hunk 2]
```diff
@@ -1858,6 +1860,7 @@ This endpoint returns an object with the following fields and a `200` code on a
 | `hostname`   | string | The hostname of the Server |
 | `id`   | string | The UUID corresponding to the Server |
 | `instance_details`   | object | Information that the cloud provider provides about the Server, if one exists |
+| `labels`   | object | (Optional) The labels for this server. Only available with the PolicySync feature, which is currently in EA |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md:1863]:
```suggestion
| `labels`   | object | (Optional) The labels for this server. This parameter is only available with the PolicySync feature, which is currently in EA. |
```

File: packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md [hunk 3]
```diff
@@ -1951,6 +1954,7 @@ This endpoint returns an object with the following fields and a `200` code on a
...
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md:1957]:
```suggestion
| `labels`   | object | (Optional) The labels for this server. This parameter is only available with the PolicySync feature, which is currently in EA. |
```

File: packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md [hunk 4]
```diff
@@ -2044,5 +2048,57 @@ https://app.scaleft.com/v1/teams/${team_name}/projects/${project_name}/servers/$
 ```json
 HTTP 204 No Content
 ```
+### Update a Server on a Project
+
+<ApiOperation method="PUT" url="https://app.scaleft.com/v1/teams/${team_name}/projects/${project_name}/servers/${server_id}" />
+Update a Server on a Project. This endpoint is only available with the PolicySync feature, which is currently in EA
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md:2054]:
```suggestion
Updates a Server on a Project. This endpoint is only available with the PolicySync feature, which is currently in EA.
```

File: packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md [hunk 5]
```diff
@@ -2044,5 +2048,57 @@ https://app.scaleft.com/v1/teams/${team_name}/projects/${project_name}/servers/$
...
+
+
+#### Request query parameters
+
+This endpoint has no query parameters.
+
+#### Request body
+
+
+This endpoint requires an object with the following fields.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md:2074]:
```suggestion
This endpoint requires an object with the following fields:
```

File: packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md [hunk 6]
```diff
@@ -2044,5 +2048,57 @@ https://app.scaleft.com/v1/teams/${team_name}/projects/${project_name}/servers/$
...
+| Properties | Type        | Description          |
+|----------|-------------|----------------------|
+| `labels`   | object | (Optional) A map of key value pairs. These labels will overwrite all labels previously supplied via the API for this server. Labels from any other source can only be updated using that source. The prefix 'api.' will be automatically prepended if not supplied. |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md:2077]:
```suggestion
| `labels`   | object | (Optional) A map of key value pairs. These labels overwrite all labels previously supplied through the API for this server. You can only update labels from other sources using that source. If you don't supply the prefix 'api.', it is automatically prepended. |
```

File: packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md [hunk 7]
```diff
@@ -77,6 +78,7 @@ This endpoint returns a list of objects with the following fields and a `200` co
 | `hostname`   | string | The hostname of the Server |
 | `id`   | string | The UUID corresponding to the Server |
 | `instance_details`   | object | Information that the cloud provider provides about the Server, if one exists |
+| `labels`   | object | (Optional) The labels for this server. Only available with the PolicySync feature, which is currently in EA. |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md:81]:
```suggestion
| `labels`   | object | (Optional) The labels for this server. This parameter is only available with the PolicySync feature, which is currently in EA. |
```

File: packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md [hunk 8]
```diff
@@ -155,6 +157,57 @@ https://app.scaleft.com/v1/teams/${team_name}/servers
 	]
 }
 ```
+### Update a Server
+
+<ApiOperation method="PUT" url="https://app.scaleft.com/v1/teams/${team_name}/servers/${server_id}" />
+Update a Server. This endpoint is only available with the PolicySync feature, which is currently in EA
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md:163]:
```suggestion
Updates a Server. This endpoint is only available with the PolicySync feature, which is currently in EA.
```

File: packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md [hunk 9]
```diff
@@ -155,6 +157,57 @@ https://app.scaleft.com/v1/teams/${team_name}/servers
...
+
+
+#### Request query parameters
+
+This endpoint has no query parameters.
+
+#### Request body
+
+
+This endpoint requires an object with the following fields.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md:182]:
```suggestion
This endpoint requires an object with the following fields:
```

File: packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md [hunk 10]
```diff
@@ -155,6 +157,57 @@ https://app.scaleft.com/v1/teams/${team_name}/servers
...
+| Properties | Type        | Description          |
+|----------|-------------|----------------------|
+| `labels`   | object | (Optional) A map of key value pairs. These labels will overwrite all labels previously supplied via the API for this server. Labels from any other source can only be updated using that source. The prefix 'api.' will be automatically prepended if not supplied. |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md:185]:
```suggestion
| `labels`   | object | (Optional) A map of key value pairs. These labels overwrite all labels previously supplied through the API for this server. You can only update labels from other sources using that source. If you don't supply the prefix 'api.', it is automatically prepended. |
```

**Review by susanharper-okta — COMMENTED** [review]
a couple of minor things. thanks.

File: packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md (hunk 10 above)
Comment (phoebeyu-okta) [reply packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md]: @susanharper-okta thanks for the review. I've addressed your comments in 3f3fbd5

**Review by susanharper-okta — APPROVED** [review]
LGTM 🚢

**Comment by phoebeyu-okta** [comment]
> LGTM @phoebeyu-okta - when is this being released?

@susanharper-okta  The feature itself will be in EA for a bit yet, but the API is available for the customers with the feature flag as of last Wednesday

**Comment by phoebeyu-okta** [comment]
@susanharper-okta Merge at will, we're fine with this going out at any point

**Pull request merged**